src/test/java/jimmy/mcgymmy/model/food/FoodTest.java

```diff
@@ -28,27 +28,51 @@
     @Test
     public void constructor_nullProtein_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () ->
-                new Food(VALID_FOOD_NAME, null, CARBOHYDRATE, FAT));
+            new Food(VALID_FOOD_NAME, null, CARBOHYDRATE, FAT));
     }
 
     @Test
     public void constructor_nullCarbohydrate_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () ->
-                new Food(VALID_FOOD_NAME, PROTEIN, null, FAT));
+            new Food(VALID_FOOD_NAME, PROTEIN, null, FAT));
     }
 
     @Test
     public void constructor_nullFat_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () ->
-                new Food(VALID_FOOD_NAME, PROTEIN, CARBOHYDRATE, null));
+            new Food(VALID_FOOD_NAME, PROTEIN, CARBOHYDRATE, null));
     }
 
     @Test
     public void constructor_invalidName_throwsIllegalArgumentException() {
         assertThrows(IllegalArgumentException.class, () ->
-                new Food(INVALID_FOOD_NAME, PROTEIN, CARBOHYDRATE, FAT));
+            new Food(INVALID_FOOD_NAME, PROTEIN, CARBOHYDRATE, FAT));
     }
 
+    @Test
+    public void toStringIsCorrect() {
```

Comment:
I think naming it `toString` is enough

Reply:
i think `toString` might override the java object's default method, so that might cause problems (including confusion in developers)

maybe we could use the module convention `toString_isCorrect` or something instead

Reply:
^^ Same comment as Joey